### DISCUSSION TITLE
Issue with color

### DISCUSSION THREAD

**Discussion by CBID2** [devicons/devicon#1476] [Q&A] [answered]
Hi there! I’ve noticed that the Markdown and GitHub icons do not appear on my profile’s README file when the website is in dark mode. Is there a way I change their colors?
Here’s a link to my file for reference: [https://github.com/CBID2/CBID2/blob/main/README.md](url)
Update: Solved it!

**Comment by Kernix13** [comment]
I cam to ask the same thing. My Markdown, CodePen, Git Bash, and Express.js do not show up in dark mode. But since you posted this on Oct 22, this discussion tab is either not checked a lot, or fixing those devicons are not a priority.

**Comment by Snailedlt** [comment, marked as answer]
Please see #977 and #1383 for more info on dark/light mode icons.

For now you can use @lunatic-fox's awesome API to show dark/light icons based on the github theme -> https://github.com/devicons/devicon/issues/1383#issuecomment-1281809031
